docs/UserGuide.md

```diff
@@ -87,87 +100,119 @@ Now that you are familiar with Reache's interface, we recommend visiting [Gettin
 
 ### 3.1 Managing Contacts
 #### 3.1.1 Adding a contact: `add`
-Adds a person to the contact list.
+Add a person to your contact list by specifying details such as:
+- Name
+- Job title
+- Company
+- Tag(s)
+- Pronoun(s)
+- Phone number(s) 
+- Email id(s)
```

Comment:
I do not think it is accurate to say id and instead it should be addresses. In most definitions, id refers to the text in the email before the @domain and just inputting the id without the domain in Reache will call out an error message.

Reply:
There is also another usage of email id in the summary table at the bottom so i think maybe thats why it was also copied here